src/client/index.ts

```diff
@@ -657,3 +745,31 @@ export function btreeItemToAggregateItem<K extends Key, ID extends string>({
     sumValue: s,
   };
 }
+
+export type NamespacedArgs<Args, Namespace> =
+  Args & { namespace: Namespace } | (Namespace extends undefined ? Args : never);
```

Comment:
so you can provide namespace: undefined if you didn't specify one? I wonder if this works if that's not intended:
```ts
Args & (Namespace extends undefined ? object : { namespace: Namespace });
```

Reply:
That is intended. It's useful for methods like `min` that call other methods and pass in the namespace.